pylint_odoo/checkers/modules_odoo.py

```diff
@@ -77,6 +78,12 @@
         'wrong-tabs-instead-of-spaces',
         settings.DESC_DFLT
     ),
+    'W%d50' % settings.BASE_OMODULE_ID: (
+        'Odoo addons relative import, should use "." '
```

Comment:
A more explanatory text should be: "Same Odoo module relative import. You should use..."